### PR TITLE
Update identity verification labs

### PR DESCRIPTION
**Pull request by DanyStinson**
*Issue #, if available:*

*Description of changes:*
We have updated the Immersion Day removing some old notebooks and adding new content. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 1]
```diff
@@ -0,0 +1,1463 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4f8f79ef-2339-4229-bbe0-b10db937c45c",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "availableInstances": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]: I don't think we need this portion in the metadata - can you check the other labs in the folder and keep this consistent?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 2]
```diff
@@ -0,0 +1,1463 @@
...
+   "execution_count": null,
+   "id": "42c7a93c-ac1d-4fee-8d70-a2ccdb3e86de",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def create_collection(collection_id):\n",
+    "    #Create a collection\n",
+    "    print('Creating collection:' + collection_id)\n",
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]: Removing redundant comment.
```suggestion
    "    print('Creating collection:' + collection_id)\n",
```

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 3]
```diff
@@ -0,0 +1,1463 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65146c9c-ff06-4a1d-8c07-036382b2ae44",
+   "metadata": {},
+   "source": [
+    "Let's display the collections in our account to verify the previous collection was completed correctly."
+   ]
+  },
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
  {
   "cell_type": "markdown",
   "id": "958fc431-f505-4c15-942b-b4d3894a6dc8",
   "metadata": {},
   "source": [
    "### Confirm your collection creation\n",
    "Let's display the collections in our account to verify the previous collection was completed correctly."
   ]
  },
```

Should we merge these two cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 4]
```diff
@@ -0,0 +1,1463 @@
...
+    "def list_collections():\n",
+    "\n",
+    "    max_results=10\n",
+    "\n",
+    "    print('Displaying collections...')\n",
+    "    response=rek_client.list_collections(MaxResults=max_results)\n",
+    "    collection_count=0\n",
+    "    done=False\n",
+    "\n",
+    "    while done==False:\n",
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "    while not done:\n",
```
nit: This reads more pythonic

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 5]
```diff
@@ -0,0 +1,1463 @@
...
+   "source": [
+    "### Create a new user"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0b996ca-e4b9-4bde-8fbf-f063a7b57cb5",
+   "metadata": {},
+   "source": [
+    "Once the collection is created, we can proceed to create a user. We are going to use the **create_user** method, which creates a new user in a collection and returns a unique user ID."
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Create a new user\n",
    "Once the collection is created, we can proceed to create a user. We are going to use the **create_user** method, which creates a new user in a collection and returns a unique user ID."
```

Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 6]
```diff
@@ -0,0 +1,1463 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a0ee3d54-d5ae-4162-8186-cb23e4cdae78",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "user2_id = \"John\"\n",
```
Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 7]
```diff
@@ -0,0 +1,1463 @@
...
+   "metadata": {},
+   "source": [
+    "### Confirm your collection creation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c60221b9-ce48-48f2-a8cb-0b8a8cee885a",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Confirm your collection creation\n",
```

Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 8]
```diff
@@ -0,0 +1,1463 @@
...
+    "### Add faces to a collection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ef2a80d-bfaf-4796-9aea-81ca1886de92",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Add faces to a collection\n",
```

Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 9]
```diff
@@ -0,0 +1,1463 @@
...
+   "metadata": {},
+   "source": [
+    "### List faces in the collection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5fab09cc-bb62-4373-976c-baf33b7ab4d0",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### List faces in the collection\n",
```

Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 10]
```diff
@@ -0,0 +1,1463 @@
...
+    "### Search face by image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7bf5f4f5-de0c-4912-9a5e-f38824ce5845",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Search face by image\n",
```

Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 11]
```diff
@@ -0,0 +1,1463 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "63e943ab-60dc-440d-8c15-b2b4271448bc",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "results = search_face_by_image(data,collection_id)[\"FaceMatches\"][0]\n",
+    "print(\"The similarity seaarching against a single low quality image is: {}\".format(results[\"Similarity\"]))"
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "print(\"The similarity searching against a single low quality image is: {}\".format(results[\"Similarity\"]))"
```

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 12]
```diff
@@ -0,0 +1,1463 @@
...
+   "metadata": {},
+   "source": [
+    "### Associate faces in the collections to a user"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7c5bf55a-38f1-4306-8e8f-e52508c546c8",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Associate faces in the collections to a user\n",
```
Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 13]
```diff
@@ -0,0 +1,1463 @@
...
+    "#### Associate a single face to a user"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c4d367e-f488-492d-8f5c-19f6aafde91d",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "#### Associate a single face to a user\n",
```
Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 14]
```diff
@@ -0,0 +1,1463 @@
...
+   "metadata": {},
+   "source": [
+    "#### Associate multiple faces in the collections to a user"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4e07c3bb-2780-4c1a-870d-a2df17088ce2",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "#### Associate multiple faces in the collections to a user\n",
```
Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 15]
```diff
@@ -0,0 +1,1463 @@
...
+    "### Search Users by UserId or FaceId"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5f70f79-012d-4ec1-bef1-ee843bd352dc",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Search Users by UserId or FaceId\n",
```
Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 16]
```diff
@@ -0,0 +1,1463 @@
...
+    "### Search Users by Image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "731267b8-f550-4d2c-b1c4-438b902fe598",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Search Users by Image\n",
```
Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 17]
```diff
@@ -0,0 +1,1463 @@
...
+    "### Disassociate faces from a user"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8f0223b-df14-413e-bc6e-64c1c3d4ab67",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Disassociate faces from a user\n",
```
Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 18]
```diff
@@ -0,0 +1,1463 @@
...
+    "### Delete a user"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb9c127b-493a-4230-ab06-dc328aeb4bde",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Delete a user\n",
```
Should we merge these cells?

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 19]
```diff
@@ -0,0 +1,1463 @@
...
+    "### Delete collection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5de00482-e5ad-4ec9-bfb1-46ec811e196c",
+   "metadata": {},
+   "source": [
```
Comment (matteofigus) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "### Delete collection\n",
```
Should we merge these cells?

**Review by matteofigus — CHANGES_REQUESTED** [review]
I added some small comments for some improvements. Also, In your PR you are adding a new 3.2 but not removing the old 3.2. Can you add that to the PR, otherwise we'll end up with two 3.2 modules.

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb (hunk 1 above)
Comment (DanyStinson) [reply identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]: Correct, this is SM Studio, let's remove it.

**Comment by DanyStinson** [comment]
Accepted changes

**Review by matteofigus — APPROVED** [review]

File: identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb [hunk 20]
```diff
@@ -0,0 +1,774 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ac08761b-341a-4e8b-8ae8-d44e08e593af",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#ListUsers - Lists the users in a collection.\n",
```
Comment (ConnorKirk) [inline identity-verification/module_3/3.2 Improving accuracy using user vectors.ipynb]:
```suggestion
    "# ListUsers - Lists the users in a collection.\n",
```

**Review by ConnorKirk — APPROVED** [review]
Added one suggestion

**Pull request merged**